startup/gui/layouts.py

```diff
@@ -65,10 +65,10 @@
-layouts.add( "Scene", "GafferUI.CompoundEditor( scriptNode, children = ( GafferUI.SplitContainer.Orientation.Horizontal, 0.772595, ( ( GafferUI.SplitContainer.Orientation.Horizontal, 0.256020, ( {'tabs': (GafferSceneUI.HierarchyView( scriptNode ),), 'tabsVisible': True, 'currentTab': 0, 'pinned': [False]}, ( GafferUI.SplitContainer.Orientation.Vertical, 0.500726, ( ( GafferUI.SplitContainer.Orientation.Vertical, 0.905605, ( {'tabs': (GafferUI.Viewer( scriptNode ), GafferSceneUI.UVInspector( scriptNode )), 'tabsVisible': True, 'currentTab': 0, 'pinned': [False, False]}, {'tabs': (GafferUI.Timeline( scriptNode ),), 'tabsVisible': False, 'currentTab': 0, 'pinned': [None]} ) ), {'tabs': (GafferUI.GraphEditor( scriptNode ), GafferUI.AnimationEditor( scriptNode ), GafferSceneUI.PrimitiveInspector( scriptNode )), 'tabsVisible': True, 'currentTab': 0, 'pinned': [None, False, False]} ) ) ) ), ( GafferUI.SplitContainer.Orientation.Vertical, 0.500726, ( {'tabs': (GafferUI.NodeEditor( scriptNode ),), 'tabsVisible': True, 'currentTab': 0, 'pinned': [False]}, {'tabs': (GafferSceneUI.SceneInspector( scriptNode ),), 'tabsVisible': True, 'currentTab': 0, 'pinned': [False]} ) ) ) ), windowState = {'fullScreen': False, 'screen': -1, 'bound': imath.Box2f(imath.V2f(0.0479166657, 0.108269393), imath.V2f(0.782812476, 0.906223357)), 'maximized': True} )" )
+layouts.add( 'Standard', "GafferUI.CompoundEditor( scriptNode, _state={ 'children' : ( GafferUI.SplitContainer.Orientation.Vertical, 0.953488, ( ( GafferUI.SplitContainer.Orientation.Horizontal, 0.699857, ( ( GafferUI.SplitContainer.Orientation.Vertical, 0.479580, ( {'tabs': (GafferUI.Viewer( scriptNode ), GafferSceneUI.UVInspector( scriptNode )), 'tabsVisible': True, 'currentTab': 0, 'pinned': [False, False]}, {'tabs': (GafferUI.GraphEditor( scriptNode ), GafferUI.AnimationEditor( scriptNode ), GafferSceneUI.PrimitiveInspector( scriptNode )), 'tabsVisible': True, 'currentTab': 0, 'pinned': [None, False, False]} ) ), ( GafferUI.SplitContainer.Orientation.Vertical, 0.539090, ( {'tabs': (GafferUI.NodeEditor( scriptNode ), GafferSceneUI.SceneInspector( scriptNode )), 'tabsVisible': True, 'currentTab': 0, 'pinned': [False, False]}, {'tabs': (GafferSceneUI.HierarchyView( scriptNode ), GafferUI.PythonEditor( scriptNode )), 'tabsVisible': True, 'currentTab': 0, 'pinned': [False, None]} ) ) ) ), {'tabs': (GafferUI.Timeline( scriptNode ),), 'tabsVisible': False, 'currentTab': 0, 'pinned': [None]} ) ), 'detachedPanels' : (), 'windowState' : { 'fullScreen' : False, 'screen' : -1, 'bound' : imath.Box2f( imath.V2f( 0.0473958328, 0.108751059 ), imath.V2f( 0.781770825, 0.906542063 ) ), 'maximized' : True }, 'editorState' : {'c-0-1-1-0-0': {'driver': 'c-0-0-0-0-0', 'driverMode': 'NodeSet'}, 'c-0-1-0-0-1': {'driver': 'c-0-1-1-0-0', 'driverMode': 'NodeSet'}, 'c-0-0-1-0-2': {'driver': 'c-0-1-1-0-0', 'driverMode': 'NodeSet'}, 'c-0-0-1-0-1': {'driver': 'c-0-1-1-0-0', 'driverMode': 'NodeSet'}} } )" )
```

Comment:
> This seems to be set up so that the HierarchyView follows the Viewer, and then the AnimationEditor, SceneInspector and PrimitiveInspector follow the HierarchyView. Questions :
> 
> * Why not just link everything directly to the viewer? It took me a while to figure out what was happening, and why unlinking the hierarchy view didn't do what I expected.

A common practice in production is to 'pin off' the viewer, to avoid display overhead. If everything was pinned to the viewer, then those editors all now need re-pining to be usable.

The idea was that when the viewer is pinned off, the Hierarchy View is now the authoritative source of scene selection, etc...  so following that is the next logical thing. With this setup, if you just unlink the hierarchy view from the viewer when its pinned off, there is minimal house keeping.

> * Should the AnimationEditor really follow the Viewer? As an _editor_, I would expect it to follow the node selection, just as the NodeEditor does.

Took a punt on that one, can easily unlink :)

Reply:
> A common practice in production is to 'pin off' the viewer, to avoid display overhead.

I think this dates from the days before the pause button, which I think is a more direct of expressing the user's intention there. I'd like to encourage folks to use the pause button instead, and if that leads to some requests to improve the pause behaviour, then so be it (the obvious one being making it more obvious when we're paused). Anyone want to argue the opposite?

> With this setup, if you just unlink the hierarchy view from the viewer when its pinned off, there is minimal house keeping.

Yeah, I get it. I just think it's more complex than necessary for newcomers, particularly those who may not have the "pin off" habit.

> Took a punt on that one, can easily unlink :)

Cool - I think that would be best.

Reply:
https://github.com/GafferHQ/gaffer/pull/3360/commits/01683b6f8ab19f25f27f440501135dca3b8fa846 Removes animation link, and links all to the viewer. This may highlight that we need to make the pause button (as that should remove the need to pin-off the viewer), but its the right direction to be going in.

Reply:
I've made https://github.com/GafferHQ/gaffer/pull/3390 to make paused viewers more obvious, which hopefully goes some way to making this the right decision.

The Multi-Monitor layout doesn't have the SceneInspector linked to the Viewer though - can we fix that?